Source/WebKit/UIProcess/API/APIWebsitePolicies.cpp

```diff
@@ -38,37 +38,37 @@ WebsitePolicies::WebsitePolicies() = default;
 Ref<WebsitePolicies> WebsitePolicies::copy() const
 {
     auto policies = WebsitePolicies::create();
-    policies->m_contentExtensionEnablement = m_contentExtensionEnablement;
-    policies->m_activeContentRuleListActionPatterns = m_activeContentRuleListActionPatterns;
-    policies->setAllowedAutoplayQuirks(m_allowedAutoplayQuirks);
-    policies->setAutoplayPolicy(m_autoplayPolicy);
+    policies->m_data.contentExtensionEnablement = m_data.contentExtensionEnablement;
```

Comment:
Can we use the WebsitePoliciesData copy constructor and just replace all this with policies->m_data = m_data; ?